compact/tracking/tof_endcap.xml

```diff
@@ -133,71 +133,48 @@
-          <slice name="Epoxy"       material="Epoxy"            thickness="SS_Epoxy_thickness"      width="SS_Epoxy_width"      length="SS_Epoxy_length"      offset="SS_Epoxy_offset"      sensitive="no"/>
-          <slice name="ALNTop"      material="AluminumNitrate"  thickness="SS_ALNTop_thickness"     width="SS_ALNTop_width"     length="SS_ALNTop_length"     offset="SS_ALNTop_offset"     sensitive="no"/>
+          <detector
+        id="ForwardTOF_ID"
+        name="ForwardTOF"
+        type="epic_TrapEndcapTracker"
+        actsType="endcap"
+        readout="TOFEndcapHits"
+        vis="TOFVis"
+        reflect="false">
+        <type_flags type="DetType_TRACKER + DetType_ENDCAP"/>
+        <module name="ForwardModule" vis="TOFModuleVis">
+          <trd x1="ForwardTOFMod_x1/2.0" x2="ForwardTOFMod_x2/2.0" z="ForwardTOFMod_y/2"/>
+          <comment> TRDs are built back-to-front </comment>
+          <module_component thickness="TOFEndcapService_thickness" material="Silicon" vis="TOFVis"/>
+          <module_component thickness="TOFEndcapSensor_thickness" material="Silicon" sensitive="true" vis="TOFSensorVis"/>
+        </module>
+        <layer id="1">
+          <envelope  vis="TOFLayerVis"
+            rmin="ForwardTOFLayer_rmin"
+            rmax="ForwardTOFLayer_rmax"
+            length="TOFEndcapLayer_thickness"
+            zstart="ForwardTOFLayer1_zmin" />
+          <ring
+            r="ForwardTOFMod_rmin + ForwardTOFMod_y/2.0"
+            zstart="0"
+            nmodules="TOFEndcapMod_count"
+            dz="TOFEndcapMod_dz"
+            module="ForwardModule" />
         </layer>
-      <layer id="3" name="ServiceHybridStack" thickness="SH_thickness" numslices="4">
-          <slice name="ThermalPad"      material="Graphite"     thickness="SH_TP_thickness"     width="SH_TP_width"         length="SH_TP_length"         offset="SH_TP_offset"         sensitive="no"/>
-          <slice name="HighSpeedBoard"  material="Polystyrene"  thickness="SH_HSB_thickness"    width="SH_HSB_width"        length="SH_HSB_length"        offset="SH_HSB_offset"        sensitive="no"/>
-          <slice name="ConnectorSpace"  material="Air"          thickness="SH_ConnSp_thickness" width="SH_ConnSp_width"     length="SH_ConnSp_length"     offset="SH_ConnSp_offset"     sensitive="no"/>
-          <slice name="Powerboard"      material="Polystyrene"  thickness="SH_PB_thickness"     width="SH_PB_width"         length="SH_PB_length"         offset="SH_PB_offset"         sensitive="no"/>
-        </layer>
-    </detector>
+      </detector>
   </detectors>
 
   <plugins>
     <plugin name="DD4hep_ParametersPlugin">
       <argument value="ForwardTOF"/>
-      <argument value="layer_pattern: str=layer_\d"/>
+      <argument value="layer_pattern: str=layer\d"/>
```

Comment:
@rahmans1 Does this line change look right?

@nschmidtALICE Was this wrong or why the understore change?

Reply:
This change is correct. The naming is different for the detailed and the "basic" TOF implementation.